src/components/vacation-requests-table/vacation-requests-table.tsx

```diff
@@ -50,11 +66,9 @@ const VacationRequestsTable = ({
    * Set data grid rows
    */
   useMemo(() => {
-    if (vacationRequests.length && vacationRequestStatuses.length) {
-      const createdRows = createDataGridRows(vacationRequests, vacationRequestStatuses);
-      if (createdRows) {
-        setRows(createdRows);
-      }
+    const createdRows = createDataGridRows(vacationRequests, vacationRequestStatuses);
+    if (createdRows) {
+      setRows(createdRows);
```

Comment:
Why not just set `createDataGridRows` into `setRows`?

Reply:
True